### DISCUSSION TITLE
Next.config.js

### DISCUSSION THREAD

**Discussion by paula-espana-fascinante** [vercel/next.js#16965] [Help] [answered]
Hi there! I'm just starting a project with Next.js as my very first try. 
I've used cmd-wordpress/canary as example. I need to add a next.config.js for images and svg loading and I'm not sure about how to config it. I also got an error when building.
When 
`npm run build`
I got this error message 
`ModuleNotFoundError: Module not found: Error: Can't resolve '../../lib/api'`
I've tried everything... As I said, I've used cmd-wordpress/canary  example. 
Thank you very much!!

**Comment by jamesmosier** [comment, marked as answer]
Hi @paula-espana-fascinante!

For your first issue (adding image & SVG loading), you could use [next-images](https://github.com/twopluszero/next-images) which is a community plugin that allows you to use inline images in your components:

```js
// next.config.js
const withImages = require('next-images')
module.exports = withImages()
```


---

For your second issue, do you have any more info you could share? That error means that it can't find the `lib/api.js` module. Which file and folder are you referencing that from?

**Reply by paula-espana-fascinante** [reply]
Thank you very much for your quick response. 
I've try next-images and works perfectly. 
Don't worry about my second issue. It works fine now. It seems that canary branch have some differences with master that I didn't notice before. The issue was about some api config.
